helix-view/src/editor.rs

```diff
@@ -981,6 +1076,7 @@ impl Editor {
         for (view, _) in self.tree.views_mut() {
             let doc = doc_mut!(self, &view.doc);
             view.sync_changes(doc);
+            view.gutters = config.gutters.clone();
```

Comment:
Does this (and the L938) change also fix https://github.com/helix-editor/helix/issues/3445?

Reply:
Yes, this will also address reloading of gutter-related configs. Thanks for tagging that. I had forgotten I did this :sweat_smile:.

I'll address the rest of the changes when I next have a bit of time - most likely by the end of the day.